README.md

```diff
@@ -48,61 +38,15 @@ networking or storage code.
 ### Installation
 
 ```sh
-npm install boardgame.io
+npm install @think-and-dev/cartesi-boardgame
 ```
 
-### Documentation
+### Documentation's boardgame.io
 
 Read our [Full Documentation](https://boardgame.io/documentation/) to learn how to
 use boardgame.io, and join the [community on gitter](https://gitter.im/boardgame-io/General)
 to ask your questions!
 
```

Comment:
Maybe a section or a comment stating the Cartesi Grant Program support would be a nice addition.